fuel-p2p/src/config.rs

```diff
@@ -0,0 +1,21 @@
+use libp2p::{Multiaddr, PeerId};
+use std::net::IpAddr;
+
+#[derive(Clone, Debug)]
+pub struct P2PConfig {
+    /// Name of the Network
+    pub network_name: String,
+
+    /// IP address for Swarm to listen on
+    pub address: IpAddr,
+
+    /// The TCP port that Swarm listens on
+    pub tcp_port: u16,
+
+    // `DiscoveryBehaviour` related fields
+    pub predefined_nodes: Vec<(PeerId, Multiaddr)>,
```

Comment:
I think it's more common to call these ["bootstrap nodes"](https://en.wikipedia.org/wiki/Bootstrapping_node)